src/locales/en.json

```diff
@@ -111,13 +111,14 @@
       "general_guidance": "General Guidance",
       "ha_guidance_header": "What should I do?",
       "header": "Someone you were nearby has since tested positive for COVID-19.",
-      "isolate": "Isolate",
+      "isolate": "Quarantine",
       "next_steps": "Next steps",
       "personalize_my_guidance": "Personalize my guidance",
       "schedule_callback": "Schedule a call to get support from a contact tracer from the {{healthAuthorityName}}.",
       "speak_with_contact_tracer": "Speak with a contact tracer",
       "wash_your_hands": "Wash your hands",
-      "wear_a_mask": "Wear a mask"
+      "wear_a_mask": "Wear a mask",
+      "quarantine": "Quarantine"
```

Comment:
If this isnt being used in the codebase, the key will be stripped during the ci process, so we will have to update the call sites as well to have this.